packages/netlify/src/index.ts

```diff
@@ -455,12 +455,7 @@ export default function netlifyIntegration(
 						hybridOutput: 'stable',
 						staticOutput: 'stable',
 						serverOutput: 'stable',
-						assets: {
-							// keeping this as experimental at least until Netlify Image CDN is out of beta
-							supportKind: 'experimental',
```

Comment:
It makes no sense for Netlify to use this to reflect the Image CDN service being experimental. This leads to a wonky spooky messages for users (assets are not supported, what does that mean?)